v2/bigtable-changestreams-to-hbase/README.md

```diff
@@ -8,17 +8,33 @@ The [BigtableToHBase](src/main/java/com/google/cloud/teleport/v2/templates/Bigta
 * Bigtable table with change streams enabled
 * Hbase instance that is accessible from Dataflow
 
-#### Beware
+#### Beware - provision Hbase resources to handle Dataflow traffic
 
 * Dataflow when strongly provisioned can translate Bigtable input QPS directly to Hbase, the user should configure Hbase so that Hbase can handle that QPS without crashing.
+
 * Bigtable change streams can handle very large single mutations. Hbase write buffers should be adequately provisioned so as not to cause exceptions when writing in large mutations.
 
+#### Beware - out-of-order writes can occur in some scenarios
+
+The replicator writes to Hbase as a side effect in Dataflow. If Dataflow loses connection to a worker, another worker will be spun up and a change stream mutation could be duplicated. In the following scenario, a duplication could occur:
+  * Dataflow spins up worker A
+  * Worker A replicates a Put
```

Comment:
I think we should call out somewhere that all mutations coming here have a timestamp assigned, so PUT is an idempotent operation by itself.